### PR TITLE
[master] More work on Hash.php

### PR DESCRIPTION
**Pull request by bantu**
Follows #811

### CODE REVIEW

File: phpseclib/Crypt/Hash.php [hunk 1]
```diff
@@ -173,27 +170,27 @@ function setHash($hash)
...
-        return strlen($output) > $this->l ? substr($output, 0, $this->l) : $output;
+        return strlen($output) > $this->length
+            ? substr($output, 0, $this->length)
+            : $output;
     }
 
     /**
      * Returns the hash length (in bytes)
      *
-     * @access public
```
Comment (GrahamCampbell) [inline phpseclib/Crypt/Hash.php]: access was public anyway - i think we should remove these annotations on 2.x too

Comment (GrahamCampbell) [inline phpseclib/Crypt/Hash.php]: that is, for all code

**Comment by terrafrost** [comment]
Looks good to me with the exception of the public / protected thing...

**Comment by bantu** [comment]
@terrafrost Removed the commit in question.

**Comment by terrafrost** [comment]
Looks good to me!

**Pull request merged**